src/pages/home/components/Planner.tsx

```diff
@@ -0,0 +1,34 @@
+export default function Planner() {
+  return (
+    <div className="flex h-full flex-col items-center justify-center gap-2.5 rounded-[20px] bg-white shadow-base">
+      <img
+        src="/assets/logo/Logo.png"
+        alt="로고"
+        className="h-[157px] w-[157px]"
+      />
```

Comment:
@chael-in 
`alt` 속성 빼놓지 않고 기재해둔 부분 좋습니다.
추가로 `<img>` 태그에는 `width`와 `height`도 지정할 수 있는데요. 이 부분은 이미지 로드 단계에서 깜빡이는 현상을 방지하고 미리 자리를 잡는 것을 도와주는 역할을 합니다.
```suggestion
      <img
        src="/assets/logo/Logo.png"
        alt="로고"
        width={157}
        height={157}
        className="h-[157px] w-[157px]"
      />
```

명시적으로 너비, 높이 값도 지정하면 더 좋을 거 같아요!


[[참고] HTML img tag](https://www.w3schools.com/tags/tag_img.asp)